_data/simple-icons.json

```
@@ -14721,6 +14726,11 @@
             "hex": "333333",
             "source": "https://github.com/stylus/stylus-lang.com/blob/c833bf697e39e1174c7c6e679e0e5a23d0baeb90/img/stylus-logo.svg"
         },
+        {
+            "title": "Stylus Extension",
```

```suggestion
            "title": "Stylus",
            "slug": "stylus_extension",
```

We can utilise the slug field for this purpose (see example in [this section](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#optional-data) of our CONTRIBUTING.md)